src/SFML/Audio/SoundSource.cpp

```diff
@@ -63,7 +63,7 @@ void SoundSource::setVolume(float volume)
 void SoundSource::setSpatializationEnabled(bool enabled)
 {
     if (auto* sound = static_cast<ma_sound*>(getSound()))
-        ma_sound_set_spatialization_enabled(sound, enabled ? MA_TRUE : MA_FALSE);
```

Comment:
I wouldn't change this, as we should use the API variables and not assume the conversion

Reply:
What's dangerous about assuming `MA_TRUE` continues to be defined as `1`? If one day Miniaudio upgrades to a C standard that has booleans (how crazy that some libs still don't assume the presence of booleans) and `MA_TRUE` is redefined as `true` then we'll still be fine.

Reply:
It's mostly just technically more correct. Maybe one day MA_TRUE is 0 and MA_FALSE -1